parser/src/lib.rs

```diff
@@ -50,12 +50,21 @@ pub enum DisplayHint {
     Debug,
     /// `:us`, formats integers as timestamps in microseconds
     Microseconds,
+    /// `__internal_bitflags_NAME` instructs the decoder to print the flags that are set, instead of
+    /// the raw value.
+    Bitflags {
+        name: String,
+        package: String,
+        disambiguator: String,
```

Comment:
shouldn't the disambiguator  field (in this variant and the `BitFlagsKey` struct) be a `u64`? I think right now it's a stringified u64?

Reply:
In the ELF / "defmt ABI", the disambiguator field is a string. We do currently always set it to a stringified `u64`, but I'd prefer to treat it as an opaque string instead.